Firebase/Core/FIRTimestamp.m

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google
+ * Copyright 2018 Google
```

Comment:
Since this is substantially the same thing as came before, you should leave the copyright alone.

In general, you shouldn't ever modify a copyright statement.

Reply:
Done.

So the rationale seems to be:

* if a file is modified OR moved, don't touch the copyright;
* if the file is added, have a copyright with the current year;
* if a file is modified AND moved, whether to update the copyright depends on whether the file was modified in any significant way.

Does the above look correct?